InactiveIssue/GetInactiveIssue.py

```diff
@@ -0,0 +1,64 @@
+from github import Github
```

Comment:
please re-order your imports to comply with PEP8.

also use snake_case for filenames

Reply:
In my opinion output is correct, I have to output those issue in which someone is assigned from more than 60 days ,so here two case happen ,first he is assigned and not shown any activity (if yes it will act directly to issue list) and if no then it will check the last commit on issue and check it's date ,if last commit was made 60 days earlier then it will add the issue to issue list.

Reply:
Currently It's running only on coala/community repo.

Reply:
If you want I can write a script script for any further use, but only problem is of `timeline of Github api` which still needs many changes

Reply:
oh, can we have it run on all repos in the org?

Reply:
Yes we can but problem is time and rate limit as we are only limited to 5000 limit (not totally sure), it take approx 17 minutes to run on my local machine on `coala/coala` repo :sweat_smile: